### PR TITLE
implemented nan_equal

### PR DESCRIPTION
**Pull request by gdementen**
unsure it should be mentioned in the changelog. What do you think?

### CODE REVIEW

**Comment by alixdamman** [comment]
A changelog would be nice. You must update api.rst also.

**Comment by gdementen** [comment]
done

File: doc/source/api.rst [hunk 1]
```diff
@@ -520,6 +520,9 @@ Session
    :toctree: _generated/
 
    Session
+   local_arrays
+   global_arrays
+   load_example_data
```
Comment (alixdamman) [inline doc/source/api.rst]: why here?

Comment (gdementen) [reply doc/source/api.rst]: because those functions return sessions/are a way to create a session. Do you see a better place?

Comment (alixdamman) [reply doc/source/api.rst]: No, LGTM.

**Pull request merged**